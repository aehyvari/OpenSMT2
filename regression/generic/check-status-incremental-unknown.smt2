(set-logic QF_UF)
(set-info :status sat)
(check-sat)
(set-info :status unsat)
(push 1)
(assert false)
(check-sat)
(pop 1)
(set-info :status sat)
(check-sat)
(set-info :status unknown)
(assert true)
(check-sat)
(set-info :status unsat)
(push 1)
(assert false)
(check-sat)
(pop 1)
(set-info :status unsat)
(assert false)
(check-sat)

