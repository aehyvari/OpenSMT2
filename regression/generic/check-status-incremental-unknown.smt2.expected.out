sat
unsat
sat
sat
unsat
unsat
